src/OrchardCore.Modules/OrchardCore.Deployment/Controllers/ImportController.cs

```diff
@@ -129,7 +130,7 @@ public async Task<IActionResult> Json(ImportJsonViewModel model)
                 return Forbid();
             }
 
-            if (!model.Json.IsJson())
+            if (!model.Json.IsJson(null, JOptions.Document))
```

Comment:
Is the rest of the deployment logic using the same options to parse the document?

Reply:
yes 
https://github.com/OrchardCMS/OrchardCore/blob/3bd5ae8a7bbf1e9b2a924e6a15d4905494a49b2b/src/OrchardCore/OrchardCore.Recipes.Core/Services/RecipeExecutor.cs#L57